src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,40 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const selectedTabs = tabs.find(tab => activeTabId === tab.id) || tabs[0];
```

Comment:
The variable `selectedTabs` is misleading as it suggests multiple tabs but holds a single tab object. Consider renaming it to `selectedTab` for better clarity.